src/utils/content.ts

```diff
@@ -10,6 +10,32 @@ export interface IContentExtractorOptions {
     content?: Partial<IContentOptions>;
 }
 
+export function sanitizeContentOptions(contentOptions: Partial<IContentOptions>): IContentOptions {
+    return {
+        trimWhiteSpace: contentOptions.trimWhiteSpace ?? true,
+        preserveIndentation: contentOptions.preserveIndentation ?? false,
+        replaceNewLines: contentOptions.replaceNewLines ?? false
+    };
+}
+
+export function mapExtractorToContentOptions(extractorOptions: IContentExtractorOptions): IContentOptions {
+    let contentOptions: IContentOptions = sanitizeContentOptions({});
+
+    if (extractorOptions.content) {
+        if (extractorOptions.content.trimWhiteSpace !== undefined) {
+            contentOptions.trimWhiteSpace = extractorOptions.content.trimWhiteSpace;
+        }
+        if (extractorOptions.content.preserveIndentation !== undefined) {
+            contentOptions.preserveIndentation = extractorOptions.content.preserveIndentation;
+        }
+        if (extractorOptions.content.replaceNewLines !== undefined) {
+            contentOptions.replaceNewLines = extractorOptions.content.replaceNewLines;
+        }
+    }
```

Comment:
The same code also exists in `callExpression.ts`, but with different default values. It would be nice to have a function `getContentOptions` that takes the extractor options and the default content options and returns the final content options.

Reply:
Good point, I didn't spot this usage before. I've now introduced a `getContentOptions` method in the content utils as you suggested and incorporated it into call expressions as well.